app/views/public/api_documentation.html.erb

```diff
@@ -342,20 +342,23 @@
 <hr />
 
 <h4 id="ping-thing">Ping Thing Post</h4>
-
-<p>The ping thing endpoint allows you to push information about transaction times.</p>
-<p>Request:</p>
-<pre>
-curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d '{"time": "NN", "signature": "signature", "transaction_type": "transfer"}' https://www.validators.app/api/v1/ping-thing/:network</pre>
-<p>Parameters:</p>
-<p><strong>`time: NN`</strong> Confirmation time in miliseconds</p>
-<p><strong>`signature: string`</strong> Transaction signature</p>
-<p>
-  <strong>`transaction_type: string`</strong> A string to describe the type of transaction. e.g. 'transfer', 'defi-swap',
-  'nft-mint', or anything meaningful to the sender.
-</p>
-<p>Response:</p>
-<pre>{"status":"ok"}</pre>
+<div class="row">
+  <div class="col-sm-12">
+    <p>The ping thing endpoint allows you to push information about transaction times.</p>
+    <p>Request:</p>
+    <pre>curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d '{"time": "NN", "signature": "signature", "transaction_type": "transfer"}'
+    https://www.validators.app/api/v1/ping-thing/:network</pre>
```

Comment:
I would not add a new line here, because it makes it impossible for users to copy and paste this example (to edit with their own data of course, but I still suspect that most users copy these examples than typing them by hand).